gatorgrouper/tests/test_group_method.py

```diff
@@ -60,6 +71,25 @@ def test_group_random_extra():
     assert num_group == 2
 
 
+@given(grpsize=integers(min_value=1, max_value=3))
+@settings(verbosity=Verbosity.verbose)
+@pytest.mark.hypothesisworks
+def test_group_random2(grpsize):
+    """This hypothesis test will test the test_group_random_extra"""
+    responses = [
+        ["Nick", True, False, True, False],
+        ["Marvin", False, False, True, True],
+        ["Evin", True, True, True, False],
+        ["Nikki", True, True, False, False],
+        ["Nick", True, False, True, False],
+        ["Dan", False, True, False, True],
+    ]
+    returned_groups = group_random.group_random_group_size(responses, grpsize)
+    size_count = grpsize
+    assert len(returned_groups[0]) == size_count
+    assert grpsize == size_count
```

Comment:
This second assertion means nothing -- `size_count` was just set to be `grpsize`. You can completely remove the mention of `size_count`. You should also decide if you need to check for more than just the size of the first group (`returned_groups[0]`). Do all groups have to have the same size to pass this test? What is the intended behavior for `group_random_group_size`?

Reply:
I deleted the second assertion. I will consult with my team about about checking the size of more than just the first group.